src/main/java/pokecube/pokeplayer/tileentity/TileEntityTransformer.java

```diff
@@ -287,10 +264,11 @@ public void onWalkedOn(final Entity entityIn)
             poke.setPokemonNickname(tag.getString("oldName"));
             PokecubeCore.LOGGER.debug("Converting {} back to a human", player.getDisplayName().getString());
             tag.remove("oldName");
-            tag.remove("is_a_Player");
+//            tag.remove("is_a_player");

```

Comment:
unless you replaced where it adds this to flag as a player, this may cause some issues later if removed

Reply:
> unless you replaced where it adds this to flag as a player, this may cause some issues later if removed

here I removed that part and put info.detach();
fixed the problem where the pokecube got the block effect